requirements.txt

```diff
@@ -1,5 +1,5 @@
 # Use >= for development versions so that source builds always work
-coala>=0.7.0.dev20160518085900
+coala>=0.7.0.dev20160522150811
```

Comment:
this does not fit in this commit anyway. if its a must and you need to do it anyway, please do it in a different commit